.github/workflows/build-package.yml

```diff
@@ -35,7 +36,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Clone tools branch
-        run: git clone -b v0.8.2 --depth=1  https://github.com/citusdata/tools.git tools
+        run: git clone -b debian-bullseye-upload-support --depth=1  https://github.com/citusdata/tools.git tools
```

Comment:
I know you already knew but let's not forget to update this after merging https://github.com/citusdata/tools/pull/190